web/scripts/start-console.sh

```diff
@@ -37,8 +37,24 @@ fi
 CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
 CONSOLE_PORT=${CONSOLE_PORT:=9000}
 
+function checkClusterEndpoint() {
+    echo "Checking cluster endpoints..."         
+    previous="$(grep 'BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=' $(pwd)/scripts/env.list | cut -d= -f2-)"
+    current=$(oc whoami --show-server)
+
+    echo "Previous cluster endpoint: $previous"
+    echo "Current  cluster endpoint: $current"
+
+    if [ "$previous" == "$current" ]; then
+        echo "Cluster endpoints are the same, env.list will not be updated."
+    else
+        echo -e "Cluster endpoints are different. \nDeleting current env.list..."
+        rm scripts/env.list
+    fi
```

Comment:
What about something like this to avoid deleting the file unexpectedly, but still make it obvious when you do need to update the environment?

```suggestion
    if [ "$previous" != "$current" ]; then
        echo "Cluster endpoint has changed"
        echo "Previous cluster endpoint: $previous"
        echo "Current cluster endpoint: $current"
        echo 'Maybe you meant to run with the "-e" flag to update scripts/env.list?'
    fi
```